ns/namespace.go

```diff
@@ -94,12 +94,69 @@ func FromCompositionTags(object interface{}, current string, namespace *[]string
 		InspectEmptyContainers(AlwaysFalse))
 }
 
-// GetValueByNamespace returns value stored in struct composition.
+// GetValueByNamespace returns value stored in nested map, array or struct composition.
+func GetValueByNamespace(object interface{}, ns []string) interface{} {
+
+	if len(ns) == 0 {
+		fmt.Fprintf(os.Stderr, "Namespace length equal to zero\n")
```

Comment:
Error handling should be done using second return argument, not just error console. Eventually you can just panic. It's better option than silently operating on incorrect data.